pkg/sensors/tracing/selectors.go

```diff
@@ -101,8 +102,24 @@ func populateArgFilterMaps(
 	pinPathPrefix string,
 	outerMap *ebpf.Map,
 ) error {
+	maxEntries := uint32(0)
+	if !kernels.MinKernelVersion("5.9") {
+		for _, vm := range k.ValueMaps() {
+			nentries := uint32(len(vm))
+			if nentries > maxEntries {
+				maxEntries = nentries
+			}
+		}
+	}
```

Comment:
Can't we use:
`maxEntries := k.ValueMapMaxExtries()`
here?

Reply:
yap, thanks!